apps/api/src/app/tenant/tenant.controller.ts

```diff
@@ -73,4 +82,28 @@ export class TenantController {
       })
     );
   }
+
+  @Put('/:tenantId')
+  @ExternalApiAccessible()
+  @ApiResponse(UpdateTenantResponseDto)
+  @ApiOperation({
+    summary: 'Update tenant',
+    description: 'Update tenant under the current environment',
+  })
+  async updateTenant(
+    @UserSession() user: IJwtPayload,
+    @Param('tenantId') tenantId: string,
```

Comment:
it should be `identifier`

Reply:
Totally agree already update in the next PR 🙏